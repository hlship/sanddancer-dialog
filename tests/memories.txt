i
i
open wallet
i
brood about job
brood jade
brood on lighter

