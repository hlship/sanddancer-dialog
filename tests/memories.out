

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> i
You have your lighter, your wallet, and your emotional baggage. You're wearing
your denim jacket.

The air is frigid, and turns your breath into clouds.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> i
You have your lighter, your wallet, and your emotional baggage. You're wearing
your denim jacket.

In your emotional baggage is your grandma's stories.

> open wallet
You open your wallet, revealing your driver's license, a receipt from Big
Jimmy's, and a folded photo.

Something about the receipt from Big Jimmy's makes you want to brood about your
shit job.

> i
You have your lighter, your wallet, and your emotional baggage. You're wearing
your denim jacket.

In your emotional baggage are your shit job and your grandma's stories.

> brood about job
Juza straddles the 371 like a drunk at last call, smelly and without a plan for
the future. Shiny cars whiz by at ninety once in a while, only stopping for gas
or directions (hint: not this way). No one ever stops for the stalls selling
food and jewelry and blankets and cheap t-shirts that cluster around the dirt
turn-off onto the rez. At least, no one buying anything.

Big Jimmy shook his head when he heard you lived in Oro Oeste. Hell of a drive,
kid, he said, popping up his Lumberjacks cap to scratch the straw pate
underneath. You should get a place in Hoo-zuh. Little Jimmy'll set you up in
that trailer cross the road for almost nothin'.

Almost nothing. Yeah, exactly what you'd turn into on the cold day in hell you
move to Juza. But on the other hand, that was before you met Ocean. Now the road
back to Oro Oeste seems longer and longer every night.

> brood jade
It's not the piece of jade you want to brood about, it's grandma's stories.

> brood on lighter
Eh. You couldn't really get into a good brood about that when it's this damn
cold.

> 
